.github/workflows/deploy-wf-service.yml

```
@@ -272,3 +272,336 @@ jobs:
+              else
+                  echo "🚨 Image tag mismatch! Expected: $TARGET_IMAGE_TAG, Found: $IMAGE_TAG"
+              fi
+
+              # Wait before the next check
+              sleep 10
+          done
+          
+  # TODO Check Application Deployment Status
+  check-wf-service-status:
+    runs-on: ubuntu-latest
+    needs: [check-prisma-migrate-status,build-and-push-ee-image]
```

YAMLlint (1.35.1)

[warning] too few spaces after comma

(commas)